src/task2/db/users.js

```diff
@@ -0,0 +1,24 @@
+import uuid from 'uuid/v1';
+
+import UserEntity from '../entitites/User';
+import { deleteItemById, getItemsByPageAndLimit } from '../utils/commonUtils';
+
+const users = [];
+
+export const getUsers = (page, limit) => getItemsByPageAndLimit(users, page, limit);
+export const getUserById = (id) => users.find(user => user.id === id);
+export const addUser = (data) => {
```

Comment:
Consider exporting all the needed things as a part of the object at the bottom of the file - thus creating a single export place like an API for your module.